heron/stmgr/src/cpp/manager/stmgr-client.cpp

```diff
@@ -177,7 +177,7 @@ void StMgrClient::SendTupleStreamMessage(proto::stmgr::TupleStreamMessage2& _msg
 }
 
 void StMgrClient::HandleTupleStreamMessage(proto::stmgr::TupleStreamMessage2* _message) {
-  release(_message);
```

Comment:
Because it comes from std::allocator but returns to mempool so never goes back to std::allocator again.